src/app/apps/rup/mapa-camas/views/mapa-camas-capa/item-cama/item-cama.component.ts

```diff
@@ -92,4 +95,46 @@ export class ItemCamaComponent implements OnChanges {
         $event.stopPropagation();
         this.accionCama.emit(relacion);
     }
+
+    setDropDown(relacion, drop) {
+        this.openedDropDown = drop;
+        if (this.openedDropDown) {
```

Comment:
Se podría simplificar este if por: this.openedDropDown?.open = this.openedDropDown === drop ? true : false; de este modo te asegurarias de que si openedDropDown existe, sigue con la operacion, en caso contrario, no se realiza.

Reply:
Como se dijo anteriormente no se puede dejar de esta manera: this.openedDropDown?.open = this.openedDropDown === drop ? true : false; ya que nos arroja el siguiente error: The left-hand side of an assignment expression may not be an optional property access. Esto hace referencia a que no se puede realizar una asignación en una propiedad opcional que podría ser null o undefined!